File: app/src/main/java/com/example/dictionary/ui/main/fragmentdiolog/MyAlertDialogFragment.java
package com.example.dictionary.ui.main.fragmentdiolog;

import android.app.AlertDialog;
import android.app.Dialog;
import android.content.DialogInterface;
import android.graphics.Color;
import android.graphics.drawable.ColorDrawable;
import android.os.Bundle;
import android.view.Gravity;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.view.Window;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.DialogFragment;

import com.example.dictionary.R;
import com.example.dictionary.db.model.Words;

public class MyAlertDialogFragment extends DialogFragment {
    private static final String ARGUMENT_ORDER = "ARGUMENT_ORDER";
   static String name;
    TextView textView;
    static Bundle args;
    public static MyAlertDialogFragment newInstance(String  title, Words words) {
        MyAlertDialogFragment frag = new MyAlertDialogFragment();
        args= new Bundle();
        args.putString("title", words.component1());
        frag.setArguments(args);
        return frag;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.material_dialog, container, false);
        textView = view.findViewById(R.id.tv_one);
        textView.setText(args.getString("title"));
        return view;
    }
//
//    @NonNull
//    @Override
//    public Dialog onCreateDialog(Bundle savedInstanceState) {
//        final Dialog dialog = super.onCreateDialog(savedInstanceState);
//        final Window window = dialog.getWindow();
//        if (window != null) {
//            window.setGravity(Gravity.TOP);
//            window.setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
//            window.requestFeature(Window.FEATURE_NO_TITLE);
//        }
//        return dialog;
//    }

}

File: app/src/main/java/com/example/dictionary/ui/main/fragmentdiolog/MyAlertDialogFragment.kt
package com.example.dictionary.ui.main.fragmentdiolog

import android.app.AlertDialog
import android.app.Dialog
import android.content.DialogInterface
import android.graphics.Color
import android.graphics.drawable.ColorDrawable
import android.os.Bundle
import android.view.Gravity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.view.Window
import android.widget.TextView
import androidx.fragment.app.DialogFragment

import com.example.dictionary.R
import com.example.dictionary.db.model.Translation

class MyAlertDialogFragment : DialogFragment() {

    private var textViewOne: TextView? = null
    private var textViewTwo: TextView? = null
    private var textViewThree: TextView? = null
    private var textViewFour: TextView? = null

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.material_dialog, container, false)
        textViewOne = view.findViewById(R.id.tv_one)
        textViewTwo = view.findViewById(R.id.tv_two)
        textViewThree = view.findViewById(R.id.tv_three)
        textViewFour = view.findViewById(R.id.tv_four)

        textViewOne!!.text = args.getString("tvOne")
        textViewTwo!!.text = args.getString("tvTwo")
        textViewThree!!.text = args.getString("tvThree")
        textViewFour!!.text = args.getString("tvFour")

        return view
    }

    companion object {
        internal lateinit var args: Bundle

        fun newInstance(trans: Translation): MyAlertDialogFragment {
            val frag = MyAlertDialogFragment()
            args = Bundle()

            args.putString("tvOne", trans.wordsOne)
            args.putString("tvTwo", trans.wordTwo)
            args.putString("tvThree", trans.wordThree)
            args.putString("tvFour", trans.wordFour)

            frag.arguments = args
            return frag
        }
    }

}